connector-webhook/src/main/java/com/redhat/cloud/notifications/connector/webhook/WebhookConnectorConfig.java

```diff
@@ -10,38 +10,74 @@
 @ApplicationScoped
 public class WebhookConnectorConfig extends ConnectorConfig {
 
-    private static final String HTTPS_CONNECT_TIMEOUT_MS = "notifications.connector.https.connect-timeout-ms";
-    private static final String HTTPS_SOCKET_TIMEOUT_MS = "notifications.connector.https.socket-timeout-ms";
+    private static final String HTTP_CONNECT_TIMEOUT_MS = "notifications.connector.http.connect-timeout-ms";
+    private static final String HTTP_CONNECTIONS_PER_ROUTE = "notifications.connector.http.connections-per-route";
+    private static final String HTTP_MAX_TOTAL_CONNECTIONS = "notifications.connector.http.max-total-connections";
+    private static final String HTTP_SOCKET_TIMEOUT_MS = "notifications.connector.http.socket-timeout-ms";
     private static final String ALTERNATIVE_NAMES = "notifications.connector.alternative.names";
+    private static final String SEDA_CONCURRENT_CONSUMERS = "notifications.connector.seda.concurrent-consumers";
+    private static final String SEDA_QUEUE_SIZE = "notifications.connector.seda.queue-size";
 
-    @ConfigProperty(name = HTTPS_CONNECT_TIMEOUT_MS, defaultValue = "2500")
-    int httpsConnectTimeout;
+    @ConfigProperty(name = HTTP_CONNECT_TIMEOUT_MS, defaultValue = "2500")
+    int httpConnectTimeout;
 
-    @ConfigProperty(name = HTTPS_SOCKET_TIMEOUT_MS, defaultValue = "2500")
-    int httpsSocketTimeout;
+    @ConfigProperty(name = HTTP_CONNECTIONS_PER_ROUTE, defaultValue = "20")
+    int httpConnectionsPerRoute;
+
+    @ConfigProperty(name = HTTP_MAX_TOTAL_CONNECTIONS, defaultValue = "200")
+    int httpMaxTotalConnections;
+
+    @ConfigProperty(name = HTTP_SOCKET_TIMEOUT_MS, defaultValue = "2500")
+    int httpSocketTimeout;
 
     @ConfigProperty(name = ALTERNATIVE_NAMES, defaultValue = "ansible")
     List<String> alternativeNames;
 
+    @ConfigProperty(name = SEDA_CONCURRENT_CONSUMERS, defaultValue = "20")
```

Comment:
NOT Camel's default value: https://camel.apache.org/components/4.0.x/seda-component.html#_component_option_concurrentConsumers

This is purposely aligned with the number of HTTP connections per route.